litellm/llms/clarifai.py

```diff
@@ -245,7 +252,7 @@ def completion(
         },
     )
     if acompletion == True:
-        return async_completion(
+        return asyncio.run(async_completion(
```

Comment:
@mogith-pn can we please have this be actually async. 

Current approach will be a blocking call for users under high traffic.